pkg/apiserver/registry/generic/resource.go

```diff
@@ -39,6 +41,14 @@ func (info *ResourceInfo) WithGroupAndShortName(group string, shortName string)
 	}
 }
 
+func (info *ResourceInfo) KeyRootFunction(ctx context.Context) string {
```

Comment:
any specific reason for using Function vs Func in the names here?

Reply:
nope, i can change that

Reply:
updated in https://github.com/grafana/grafana/pull/89635/commits/ec935986be833cf71819ae871d82f4d2e3c56a66